src/main/java/md/maib/service/impl/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package md.maib.service.impl;
+
+import md.maib.dto.AuthenticationRequest;
+import md.maib.dto.AuthenticationResponse;
+import md.maib.repository.CustomerRepository;
+import md.maib.service.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AuthenticationServiceImpl.class);
+
+    private final CustomerRepository customerRepository;
+
+    @Autowired
+    public AuthenticationServiceImpl(CustomerRepository customerRepository) {
+        this.customerRepository = customerRepository;
+    }
+
+    @Override
+    public Optional<AuthenticationResponse> loginUser(AuthenticationRequest request) {
+        LOGGER.info("Login user with pan: {}", request.getPan());
+        return customerRepository.findByPan(request.getPan())
+                .filter(customer -> customer.getCvv().equals(request.getCvv()))
```

Comment:
If we use user's **cvv** as password, we have to think about security.
Make a research how to make it more secure, instead of passing it in open format.